### DISCUSSION TITLE
Should "coverage" be added to .gitignore?

### DISCUSSION THREAD

**Discussion by Adnan-Bacic** [react-native-community/discussions-and-proposals#569] [Ideas]
when running:

```
jest --coverage
```

its creates a folder called "coverage". it can easily create a few hundred files, maybe even thousands in large projects.

should this folder be added to `.gitignore`?

i can see the .gitingore file in the jest repo itself contains the "coverage" folder: https://github.com/facebook/jest/blob/main/.gitignore

and when creating a react project with `create-react-app`, it also has the "coverage" folder in .gitinore, like so:
```
# testing
/coverage
```

shouldt react-native also have the same 2 lines in `.gitingore`?

**Comment by kelset** [comment]
sure, feel free to open a PR for it! I'd say that it needs to be done in the /template folder so that new projects will benefit from it

**Comment by Adnan-Bacic** [comment]
my first contribution to the react-native repo: https://github.com/facebook/react-native/pull/35809

**Reply by kelset** [reply]
and it's already merged 🥳
thanks for your help 🤗
